### ISSUE TITLE
Plugin puts submission files for one submission in multiple folders

### ISSUE THREAD

**Issue by amandastevens** [pkp/plagiarism#3]
On a Publishing Services client's OJS 3.1.1-4 site the Ithenticate plugin is sending all submission files (including cover letters, title pages, and figures) to Ithenticate, and instead of the files being added to one folder per submission they are being added to separate folders. As a result, there are multiple folders per submission and the client is being charged for each file instead of each submission.

The client contacted Turnitin and they said, "We will process any files which are sent to us but we cannot make OJS send those files so the fault has to be within OJS." 

![ithenticate folders](https://user-images.githubusercontent.com/35044512/49612561-755b2000-f97b-11e8-993c-b90413a6a300.png)

**Comment by asmecher** [comment]
@amandastevens, this will depend on a third-party library accepting a pull request (https://github.com/bsobbe/iThenticate/pull/4/files) before I can officially add it. Once that's done I can merge this and prepare a release. In the meantime you can apply https://github.com/asmecher/plagiarism/commit/cd4096d7f5e0dc344bbddcf6bc506f2e41c3b327 and https://github.com/bsobbe/iThenticate/pull/4/files manually if it's a rush to support this for someone.

**Comment by asmecher** [comment]
@amandastevens, did you apply both patches? There's one to the plugin, and one to the library it depends on.

**Comment by amandastevens** [comment]
Oops! We only applied one patch. We've applied the 2nd one now so it should work. Sorry about that!

**Comment by drugurkocak** [comment]
Hi,
I installed this plugin and seems to work very well. But it still sends all the document files (docx, and pdf) even I check the supplementary tickbox, and set filetype groupe to artwork or supplementary content. I applied both patches above.  OJS 3.1.2-4. 
If it would select the primary submission file (which should be the blind manuscript that does not contain author names), it would be perfect for use.
Anyways, thank you for your efforts.
